### ISSUE TITLE
Needs focus styles

### ISSUE THREAD

**Issue by axlevxa** [axlevxa/typowiki#5]
For accessibility/usability, there should be a visible indicator when any interactive element is focused. Most links and buttons on this template have none.

I am thinking something along the lines of...

```
:focus {
    outline: .125rem solid #e3a122;
    outline-offset: .125rem;
}
```

**Comment by axlevxa** [comment]
Also the skip link should be visible when focused

**Comment by axlevxa** [comment]
Fixed in v1.2
